src/tagged-type.ts

```diff
@@ -127,18 +129,11 @@ export class TaggedTypeEncoded {
       for (const i in input) _out.push(TaggedTypeFormat.encode(input[i]));
       return _out;
     },
-    date: (dateValue: Date): TaggedDate | TaggedTime => {
-      if (
-        dateValue.getUTCHours() == 0 &&
-        dateValue.getUTCMinutes() == 0 &&
-        dateValue.getUTCSeconds() == 0 &&
-        dateValue.getUTCMilliseconds() == 0
-      ) {
-        return { "@date": dateValue.toISOString().split("T")[0] };
-      }
-
-      return { "@time": dateValue.toISOString() };
-    },
+    date: (dateValue: Date): TaggedTime => ({
+      "@time": dateValue.toISOString(),
+    }),
+    faunadate: (value: DateStub): TaggedDate => ({ "@date": value.dateString }),
+    faunatime: (value: TimeStub): TaggedTime => ({ "@time": value.isoString }),
   };
```

Comment:
i'd call these `date_stub` and `time_stub` for clarity but OK to ship like this for now.

Reply:
oh good catch.